fbchat/client.py

```diff
@@ -797,18 +799,26 @@ def fetchUnread(self):
         :raises: FBchatException if request failed
         """
         form = {
-            'client': 'mercury_sync',
             'folders[0]': 'inbox',
+            'client': 'mercury',
             'last_action_timestamp': now() - 60*1000
             # 'last_action_timestamp': 0
         }
 
-        j = self._post(self.req_url.THREAD_SYNC, form, fix_request=True, as_json=True)
+        j = self._post(self.req_url.UNREAD_THREADS, form, fix_request=True, as_json=True)
 
-        return {
-            "message_counts": j['payload']['message_counts'],
-            "unseen_threads": j['payload']['unseen_thread_ids']
-        }
+        return j['payload']['unread_thread_fbids'][0]['other_user_fbids']
+
+    def fetchUnseen(self):
```

Comment:
Can't verify whether `fetchUnseen` works or not, since I don't really know what it does. Add some documentation ;)

Reply:
fetchUnseen should return a list of "new" threads but I was not able to make FB return anything except an empty array :)
That request really exists though (seen in fiddler), and FB returns some json